tests/unittests/tests-hashes/tests-hashes-keccak.c

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2017 Adrian Herrmann <adrian.herrmann@fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     unittests
+ * @{
+ *
+ * @file
+ * @brief       Test cases for the Keccak hashing package
+ *
+ * @author      Adrian Herrmann <adrian.herrmann@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+#include "KeccakHash800.h"
+#include "KeccakHash1600.h"
+#include "embUnit.h"
+
+Keccak800Hash_instance* hash800Instance;
+Keccak1600Hash_instance* hash1600Instance;
+
+const BitLength databitlen = 80;
```

Comment:
(see my other comment on bit length vs byte length in RIOT)
I think you can eliminate this variable.